tests/test_unit/utils.py

```diff
@@ -464,3 +465,48 @@ def _type_checking(cls) -> dict[str, Any]:
             "NumpyFloat64Array": NumpyFloat64Array,
             "NumpyObjArray": NumpyObjArray,
         }
+
+
+class MockQuestion:
```

Comment:
these are fine, but it seems like you could equally use mock() with little downside and less code